descriptors/ModuleDescriptor-template.json

```diff
@@ -384,31 +384,34 @@
       }
     },
     "env": [
-      { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0", "description": "Java options" },
+      { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0", "description": "Java options." },
       { "name": "CAMUNDA_BPM_ADMINUSER_EMAIL", "value": "admin@localhost", "description": "The e-mail address of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_ADMINUSER_ID", "value": "admin", "description": "The account name of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_ADMINUSER_PASSWORD", "value": "admin", "description": "The password of the Camunda administration user." },
       { "name": "CAMUNDA_BPM_DATABASE_SCHEMAUPDATE", "value": "true", "description": "If Camunda should auto-update the BPM database schema." },
       { "name": "CAMUNDA_BPM_METRICS", "value": "false", "description": "Enable or disable Camunda metrics by default." },
-      { "name": "DB_HOST", "value": "postgres", "description": "Postgres hostname" },
-      { "name": "DB_PORT", "value": "5432", "description": "Postgres port" },
-      { "name": "DB_USERNAME", "value": "folio_admin", "description": "Postgres username" },
-      { "name": "DB_PASSWORD", "value": "folio_admin", "description": "Postgres username password" },
-      { "name": "DB_DATABASE", "value": "okapi_modules", "description": "Postgres database name" },
+      { "name": "DB_HOST", "value": "postgres", "description": "Postgres host name." },
+      { "name": "DB_PORT", "value": "5432", "description": "Postgres port." },
+      { "name": "DB_USERNAME", "value": "folio_admin", "description": "Postgres user name." },
```

Comment:
https://english.stackexchange.com/questions/43436/username-user-name-or-user-name

I think this is correct as `username`.